stdlib/ext/file-ext.mc

```diff
@@ -105,6 +114,21 @@ utest
   else ("Error reading file","","","")
 with ("Hello", "Next string", "Final", "EOF") in
 
+-- Test reading x amount of characters from the file
+utest
```

Comment:
Consider nesting the test cases like this:
```
  match readOpen filename with Some rc then
  	utest readBytes rc 3 with Some "Hel" using optionEq eqString in
  	utest readBytes rc 4 with Some "lo\n" using optionEq eqString in 
  	utest readBytes rc 0 with Some "" using optionEq eqString in 
  	utest readBytes rc 1 with Some "e" using optionEq eqString in 
  	utest readBytes rc 1000 with Some "xt string \nFinal" using optionEq eqString in
  	utest readBytes rc 1000 with None () using optionEq eqString in 
  	utest readBytes rc 1000 with None () using optionEq eqString in 
  	()
  else 
  	error "File could not be read in tests for readBytes"
```

Reply:
That looks better, thanks!